Integrations/integration-ArcherRSA.yml

```diff
@@ -3682,9 +3688,13 @@ script:
     - name: fieldID
       required: true
       description: Field Id
-    description: Returns a list of values for a specified field, e.g., fieldID=16114. This
-      command only works for value list fields (type 4).
+    description: Returns a list of values for a specified field, e.g., fieldID=16114.
+      This command only works for value list fields (type 4).
+  - name: archer-reset-cache
+    arguments: []
+    description: Reset Archer's integration cache
```

Comment:
maybe add a description when this should be done ?

Reply:
Updated